code/modules/integrated_electronics/subtypes/output.dm

```diff
@@ -128,7 +128,7 @@
 
 /obj/item/integrated_circuit/output/text_to_speech
 	name = "text-to-speech circuit"
-	desc = "A miniature speaker is attached to this component."
+	desc = "Takes any string as an input, and will make the device say the string when pulsed."
```

Comment:
Do these really need a comma?

Reply:
probably dont need the oxford comma here, but yeah

Reply:
yeah remove it please